<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunAllRaster</class>
 <widget class="QDialog" name="RunAllRaster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RunAllRaster</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>6</y>
     <width>571</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Input - ( Step 1 - Suggest Me! )</string>
   </property>
   <widget class="QLineEdit" name="lineEditDEM">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>34</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>37</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Digital Elevation Model (ESRI binary grid or ascii raster data)
for the modeling domain.</string>
    </property>
    <property name="text">
     <string>DEM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>77</y>
      <width>291</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Minimum Number of Grids draining to a particular Cell/Grid
to be classified as a Stream Grid.
Click Suggest Me for a initial guess!</string>
    </property>
    <property name="text">
     <string>Stream Grid Threshold (# of grids)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditThreshold">
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>75</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDEM">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSuggestMe">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>71</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&quot;Suggest Me&quot; should be executed first before &quot;Run&quot; --
Some of the files generated earlier are used when Run is executed.
However, you could change tolerance later if needed.</string>
    </property>
    <property name="text">
     <string>Suggest Me!</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>116</y>
     <width>571</width>
     <height>144</height>
    </rect>
   </property>
   <property name="title">
    <string>Output - ( Step 2 - Run )</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>43</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>File to which Stream shapefile generated by
RasterProcessing will be saved.</string>
    </property>
    <property name="text">
     <string>Stream</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>83</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>File to which Watershed boundary shapefile generated by
RasterProcessing will be saved.</string>
    </property>
    <property name="text">
     <string>Watershed</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStream">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>40</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditWatershed">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>80</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStream">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>37</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonWatershed">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>76</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxVector">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>113</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Vector Data</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRaster">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>113</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Note that loding Raster Data can take time.</string>
    </property>
    <property name="text">
     <string>Load Raster Data</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>270</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButtonHelp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>11</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRun">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>11</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&quot;Suggest Me&quot; should be executed first before &quot;Run&quot; --
Some of the files generated earlier are used when Run is executed.
However, you could change tolerance later if needed.</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="Q3TextBrowser" name="textBrowser" native="true">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>330</y>
     <width>571</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>QWidget</extends>
   <header>q3textbrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
